tests/new-smoke-all-test.ps1

```diff
@@ -45,6 +45,10 @@ _quarto:
       ensureDocxRegexMatches:
         - []
         - []
+    pptx:
+      ensurePptxRegexMatches:
```

Comment:
I don't think we have this functionality. It makes sense, and we should, but we don't. See tests/smoke/smoke-all.ts for how `ensureDocxRegexMatches` is implemented.